documentation/src/main/asciidoc/topics/con_hotrod_client_intelligence.adoc

```diff
@@ -1,10 +1,17 @@
 [id='hotrod-client-intelligence_{context}']
 = Client Intelligence
-Hot Rod clients use intelligence mechanisms to efficiently send requests to {brandname} Server clusters.
+{hr_name} clients use intelligence mechanisms to efficiently send requests to {brandname} Server clusters. 
+By default, the {hr_name} protocol has the `HASH_DISTRIBUTION_AWARE` intelligence mechanism enabled.  
 
 .`BASIC` intelligence
 Clients do not receive topology change events for {brandname} clusters, such as nodes joining or leaving, and use only the list of {brandname} Server network locations that you add to the client configuration.
 
+[NOTE]
+====
+Enable `BASIC` intelligence to expose a hidden IP address for a {hr_name} client that sent a request to a remote server. 
```

Comment:
Enable ... to use the client configuration only if the server cluster should not send back the topology as it is internal and hidden to the client. Examples are LoadBalancer, NAT, Proxy or OpenShift environment.

Reply:
Thanks, @wfink . I updated the content based on your feedback. Please let me know whay you think.